tools/db_bench_tool.cc

```diff
@@ -4176,7 +4176,8 @@ void VerifyDBFromDB(std::string& truth_db_name) {
         if (levelMeta.level == 0) {
           for (auto& fileMeta : levelMeta.files) {
             fprintf(stdout, "Level[%d]: %s(size: %" PRIu64 " bytes)\n",
-                    levelMeta.level, fileMeta.name.c_str(), fileMeta.size);
+                    levelMeta.level, fileMeta.name.c_str(),
+                    (unsigned long long)fileMeta.size);
```

Comment:
isn't fileMeta.size of type uint64_t?
https://github.com/facebook/rocksdb/blob/master/include/rocksdb/metadata.h#L84